e2e-test/src/test/resources/testdata/BigQuery/bqDateTransformCreateTableLong.txt

```diff
@@ -0,0 +1 @@
+create table `DATASET.TABLE_NAME` ( Name STRING, date_of_admission INT64)
```

Comment:
As discussed combine this in other table

Reply:
@sawantpritam Combine table and deleted extra table